orders-redis/src/test/java/io/helidon/examples/sockshop/orders/redis/TestRedisOrderRepository.java

```diff
@@ -11,7 +11,7 @@
 import org.redisson.api.RMap;
 
 @Alternative
-@Priority(Interceptor.Priority.APPLICATION+100)
+@Priority(Interceptor.Priority.APPLICATION+5)
 public class TestRedisOrderRepository extends RedisOrderRepository implements TestOrderRepository {
     @Inject
     public TestRedisOrderRepository(RMap<String, Order> carts) {
```

Comment:
Shouldn't the above parameter name be `orders` instead of `carts`?

Reply:
Maybe. The origin of the name is in the original `RedisOrderRepository` code in master:

https://github.com/helidon-sock-shop/orders/blob/master/orders-redis/src/main/java/io/helidon/examples/sockshop/orders/redis/RedisOrderRepository.java#L22-L23

Reply:
Yeah, definitely copy&paste error on my end. Do you mind fixing it in both places so I can merge it as part of this PR?

Otherwise I can merge as-is and fix it later...